### PR TITLE
room: fix subset create when group exists

### PR DESCRIPTION
**Pull request by fblackburn1**
why: it was impossible to create a chat room with subset of participants
if a bigger group existed

### CODE REVIEW

File: integration_tests/suite/test_user_room.py [hunk 1]
```diff
@@ -231,6 +232,30 @@ def test_create_minimal_parameters(self):
...
 
+    @fixtures.http.room(users=[USER_1, USER_2, USER_3])
+    def test_create_when_a_group_of_users_already_exists(self, existing_room):
+        room_args = {
+            'users': [USER_1, USER_2],
+        }
+        room = self.chatd.rooms.create_from_user(room_args)
+
+        assert_that(room['uuid'], not_(existing_room['uuid']))
+        self._delete_room(room)
```
Comment (DrPyser) [inline integration_tests/suite/test_user_room.py:243]: why the `self._delete_room`? Isn't it part of the fixture teardown?

Comment (fblackburn1) [reply integration_tests/suite/test_user_room.py:243]: This call delete the room from the `self.chatd.rooms.create_from_user` call, not from the fixture

Comment (DrPyser) [reply integration_tests/suite/test_user_room.py:243]: ah oops, my bad

**Review by DrPyser — APPROVED** [review]
Could verify the fix with manual testing(through API). if exact set of participants already exist in a room, that room is returned. otherwise a new room is created with that set of participant.

**Pull request merged**